darglint2/config.py

```diff
@@ -137,6 +159,21 @@ def log_level(self, log_level: LogLevel) -> None:
         logger = get_logger()
         logger.setLevel(log_level.value)
 
+    @property
+    def errors_to_ignore(self) -> List[str]:
+        """Update the errors to ignore, accounding for defaults.
+
+        For use in constructing a cached `errors_to_ignore` value.
+        Since this value could be used frequently, it makes
+        sense to cache this value.
+
+        Returns:
+            The errors to ignore, including the default errors.
+
+        """
+        disabled = DEFAULT_DISABLED - set(self.enable)
+        return self.ignore + list(disabled)
```

Comment:
This list may have duplicates, right?

Also, for repeatability and unit testing purposes, `sorted(disabled)` might be better?

Or change the return value to a set if order doesn't matter?